### ISSUE TITLE
Problem about different number of input views on Replica dataset

### ISSUE THREAD

**Issue by DaLi-Jack** [autonomousvision/monosdf#79]
Hi ! Thanks for the wonderful work !
I noticed that you have Figure 6 in the supplementary material discussing model performance for different input views. 
I want to know how you **select a part of the views for the same scene**. I randomly selected 30 views for Replica scan1, trained for 400 epochs, with depth, normal cues, but the result is very poor. In addition, I also want to know how many epochs are suitable for training when I reduce the input views.
1.shape reconstruction result in meshlab (400 epochs, 30 views)
![6aded4c51ff5a259a0ae6b9524e6403](https://github.com/autonomousvision/monosdf/assets/70205757/3ce5f41f-1dfc-48bc-af89-e6731d53020e)
2.rendering results (400 epochs, 30 views)
![4000e36b29185ed59f3362844c24688](https://github.com/autonomousvision/monosdf/assets/70205757/ae8cf311-99b2-42bf-b22f-688345a28b45)
Any help would be greatly appreciated !

**Comment by niujinshuchong** [comment]
Hi, I think we trained for the same epochs with using 100 views in the replica dataset. Can you get good results with all the views (100)?

**Comment by DaLi-Jack** [comment]
Yes ! I have already reproduced your results (using 100 views, with training 400 epochs), and that's very good !
But I want to further test the effect of the algorithm after reducing the views, as you have shown in the supplementary material Figure 6. However, as mentioned in the previous issue, the effect is poor after reducing the views (very different from Fig. 6 in 
your supp), so I would like to ask how you choose a part of the views and how many epochs are suitable for training when I reduce the input views.
Any help would be greatly appreciated !

**Comment by niujinshuchong** [comment]
Oh, could you try on other scene of the replica dataset? I haven't test 10 views on this scene but only on the scene we show in the supp. though I think it should also work. Could you using MLP or hash grids?

**Comment by DaLi-Jack** [comment]
I used MLP. I will try it on other scenes, Thanks !

**Comment by zhanghaoyu816** [comment]
Hello, what's your results on other replica scenes when reducing the input views?
